src/components/common/verify-mfa-code/verify-mfa-code-service.ts

```diff
@@ -18,7 +19,7 @@ export function verifyMfaCodeService(
   const verifyMfaCode = async function (
     methodType: MFA_METHOD_TYPE,
     code: string,
-    isRegistration: boolean,
```

Comment:
Do ensure we have ZDD with this change, we'll have to deploy this first we'll need to keep the `isRegistration` in the request for now. We should be able to remove it after the API is deployed